src/self_update.sh

```diff
@@ -155,6 +159,11 @@ function parse_self_update_options()
 
 function self_update_help()
 {
+  if [[ "$1" == --help || "$1" == -h ]]; then
```

Comment:
Note that the file `help.sh` has a quick summary of all features; add `kw self-update` there.